packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -787,15 +787,33 @@ class RenderParagraph extends RenderBox
 
   /// Returns a list of rects that bound the given selection.
   ///
-  /// A given selection might have more than one rect if this text painter
-  /// contains bidirectional text because logically contiguous text might not be
-  /// visually contiguous.
+  /// The [boxHeightStyle] and [boxWidthStyle] arguments may be used to select
+  /// the shape of the [TextBox]es. These properties default to
+  /// [ui.BoxHeightStyle.tight] and [ui.BoxWidthStyle.tight] respectively and
+  /// must not be null.
+  ///
+  /// A given selection might have more than one rect if the [RenderParagraph]
+  /// contains multiple [InlineSpan]s or bidirectional text, because logically
+  /// contiguous text might not be visually contiguous.
+  ///
+  /// See [TextPainter.getBoxesForSelection], for which this is essentially a
```

Comment:
This can just be a `See also` section to keep it stylistically the same as the rest of the docs and we also don't want to explicitly describe the implementation details to keep API promises a blackbox. Try:

```
/// See also:
///
///  * [TextPainter.getBoxesForSelection], the method in TextPainter to get the equivalent boxes.
```

Reply:
Great -- done.